services/simcore/template.env

```diff
@@ -10,10 +10,6 @@ AGENT_VOLUMES_CLEANUP_S3_BUCKET=${AGENT_VOLUMES_CLEANUP_S3_BUCKET}
 AGENT_VOLUMES_CLEANUP_S3_PROVIDER=${AGENT_VOLUMES_CLEANUP_S3_PROVIDER}
 AGENT_VOLUMES_CLEANUP_TARGET_SWARM_STACK_NAME=${AGENT_VOLUMES_CLEANUP_TARGET_SWARM_STACK_NAME}
 
-
-
-
-
 API_DOMAIN='${API_DOMAIN}'
```

Comment:
Could ypou try to sort the lines in this file alphabetically, for example with this pre-commit hook or so? https://github.com/pre-commit/pre-commit-hooks/blob/main/pre_commit_hooks/file_contents_sorter.py

Reply:
I tried but didn't manage to make it work. I will leave it for now